load.php

```diff
@@ -23,6 +23,18 @@
 			'block-on-failing' => false,
 			'hide-column' => false,
 		],
+		'clone-republish' => [
+			'enabled' => true,
+			'post-types' => [
+				'post',
+				'page',
+			],
+			'roles' => [
+				'editor',
+				'administrator',
+			],
+			'excluded-taxonomies' => false,
```

Comment:
I think this would error out as the function expects to return an array - we're using a null coalesce in the pre option filter, given `false` is not null that filter will return `false` rather than an empty array.
```suggestion
			'excluded-taxonomies' => null,
```